test/react-native/features/app_data.feature

```diff
@@ -0,0 +1,81 @@
+Feature: App data
+
+Scenario: Handled JS error
+  When I run "AppJsHandledScenario" and relaunch the app
+  Then I wait to receive a request
+  And the exception "errorClass" equals "Error"
+  And the exception "message" equals "HandledCaughtError"
+  And the event "app.version" equals "1.2.3"
+  And the event "app.versionCode" equals 1
+  And the event "app.releaseStage" equals "production"
+  And the event "app.inForeground" is true
+  And the event "app.duration" is not null
+  And the event "app.durationInForeground" is not null
+
```

Comment:
Looking at the notifier spec, I think I would expect a few more fields to be checked in each scenario?
- binaryArch
- buildUuid (possibly)
- codeBundleId
- id
- type
- inForeground

Some may require a bit more work (e.g. setting in config) or be less easy to predict.

Reply:
- `binaryArch` is only relevant for native native (NDK/C layer stuff) – we could manually set it to ensure it gets set, but I don't know how useful that is?
- `buildUuid` is generated by our gradle plugin so would have to add that to the fixture app if we wanted it
- `codeBundleId` is straightforward to test – I'll add a case for it
- `id` already tested
- `type` I'll add a case
- `inForegound` already tested